lntest/node.go

```diff
@@ -134,7 +134,12 @@ type BackendConfig interface {
 }
 
 type NodeConfig struct {
-	Name       string
+	Name string
+
+	// LogFilenamePrefix is is used to prefix node log files. Can be used
+	// to store the current test case for simpler postmortem debugging.
+	LogFilenamePrefix string
```

Comment:
doesn't look like it is used as an actual prefix 😛

Reply:
Yeah, true, node id is the real prefix, but ignoring that, the real-real prefix the is testcase name :smile: